YakShaver Docs - Recording With Your Smartphone (PR from TinaCMS) (#478)
SSWConsulting/SSW.Products: content/docs/YakShaver/recording-work-item-smartphone.mdx

---
seo:

>   title: Recording With Your Smartphone
> title: Recording With Your Smartphone

Reviewer: ```suggestion
  title: Recording Shaves with Your Smartphone
title: Recording Shaves with Your Smartphone
```

date: '2025-09-16T22:00:00.000Z'
---

> If you ever need to record a YakShave from your smartphone (mobile app, responsive website, …), you have 2 options at your disposal

Reviewer: ```suggestion
If you ever need to record a YakShave from your smartphone (mobile app, responsive website etc.), you have 2 options at your disposal
```

## Option 1: Recording your smartphone's screen

> The process is easy, just record your screen (don't forget to include microphone audio) and then upload the recording to your Recordings OneDrive folder. Then YakShaver will pick up the video and processs it.

Reviewer: ```suggestion
Record your screen (don't forget to include microphone audio) and then upload the recording to your OneDrive's Recordings folder. YakShaver will pick up the video and process it.
```

> You can use any screen recording software but the easiest way is to use your smartphone's screen recording app.

Reviewer: ```suggestion
You can use any screen recording software, but the easiest way is to use your smartphone's screen recording app.
```

\
\
3\. Record your YakShave and then upload it to your OneDrive Recordings folder.

### iOS

1. Swipe down from the top-right corner to open the Control Center and click on the Start Screen Recording button. If you can't find it, you may have to add it there.

<imageEmbed alt="Image" size="small" showBorder={true} src="/YakShaver/Docs/IOS-screen-recorder.png" />

\
\
2\. Turn on Microphone Audio by long-pressing the record button in the control center

<imageEmbed alt="Image" size="small" showBorder={true} src="/YakShaver/Docs/IOS-screen-recorder-setup.jpg" />

\
\
3\. Record your YakShave and then upload it to your OneDrive Recordings folder.

## Option 2: Using Teams mobile app

Provided that you have already set up [Microsoft Teams](/docs/recording-work-item-teams), you can use the mobile Teams app to record a YakShave

1. Open Teams on your smartphone
2. Go to the Calendar tab and click on the meeting icon on the top-right corner

<imageEmbed alt="Image" size="small" showBorder={true} src="/YakShaver/Docs/Teams-start-meeting.png" />

\
\
3\. Start a meeting and share your screen

> 4\. Record your YakShave and it should be uploaded to your OneDrive and picked up by YakShaver

Reviewer: ```suggestion
4\. Record your YakShave and it should be uploaded to your OneDrive automatically and picked up by YakShaver.
```
